cmd/agent/subcommands/jmx/command.go

```diff
@@ -131,9 +131,6 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 			fx.Supply(params),
 			core.Bundle(),
 			diagnosesendermanagerimpl.Module(),
-			fx.Supply(func(diagnoseSenderManager diagnosesendermanager.Component) (sender.SenderManager, error) {
```

Comment:
We create the sender manager in line 315
https://github.com/DataDog/datadog-agent/blob/462f29c9ed3cfff5b6f858fcf4573c35eaddb764/cmd/agent/subcommands/jmx/command.go#L315-L318

No need to create two 😉